gtsam/sfm/sfm.i

```diff
@@ -4,6 +4,60 @@
 
 namespace gtsam {
 
+#include <gtsam/sfm/DsfTrackGenerator.h>
+
+class MatchIndicesMap {
+  MatchIndicesMap();
+  MatchIndicesMap(const gtsam::MatchIndicesMap& other);
+
+  size_t size() const;
+  bool empty() const;
+  void clear();
+  gtsam::CorrespondenceIndices at(const pair<size_t, size_t>& keypair) const;
+};
+
+
+class KeypointsList {
+  KeypointsList();
+  KeypointsList(const gtsam::KeypointsList& other);
+  void push_back(const gtsam::Keypoints& keypoints);
+  size_t size() const;
+  bool empty() const;
+  void clear();
+  gtsam::Keypoints at(const size_t& index) const;
+};
+
+
+class Keypoints
+{
+  Keypoints(const gtsam::KeypointCoordinates& coordinates);
+  gtsam::KeypointCoordinates coordinates;
```

Comment:
has KeypointCoordinates been wrapped?

Reply:
No need to wrap it, since it's just a typedef on Eigen, with implicit conversion

```c++
using KeypointCoordinates = Eigen::MatrixX2d;
```